engine/server/engine/centralized_logs/client_implementations/persistent_volume/log_file_manager/log_file_manager.go

```diff
@@ -206,25 +206,27 @@ func (manager *LogFileManager) createSymlinkLogFile(targetLogFilePath, symlinkLo
 	return nil
 }
 
-// TODO: Implement a FilePath Builder to remove exploding constructors
+// TODO: Implement a FilePath Builder to centralize log file path creation across the entire module
```

Comment:
log code is in need of a refactor to centralize all log file path creation. wanted to get this fix out but going to do this refactor in my next available cycle

Reply:
ideally i would've only had to make this change in on but i had to in several places to fix formatting